data_steward/tools/generate_unioned_ehr_dataset.sh

```diff
@@ -4,12 +4,21 @@ set -ex
 # Generate unioned_ehr dataset
 # Loads vocabulary from specified dataset and unioned EHR tables from specified dataset
 
+deactivation='false'
+pids_project_id='test_project'
+pids_dataset_id='test_dataset'
+pids_table='test_table'
+
```

Comment:
why add default values?  This nullifies the string length check in lines 67 and 68 because they are always set.